packages/wallet/src/test-utils/launch-custom-provider-and-get-wallets.ts

```diff
@@ -0,0 +1,66 @@
+import type { Provider } from '@fuel-ts/providers';
+import { setupTestProvider } from '@fuel-ts/providers/test-utils';
+import type { SetupTestProviderOptions } from '@fuel-ts/providers/test-utils';
+
+import type { WalletUnlocked } from '../wallets';
+
+import { WalletConfig } from './wallet-config';
+
+export interface LaunchCustomProviderAndGetWalletsOptions extends SetupTestProviderOptions {
+  /** Configures the wallets that should exist in the genesis block of a the test node. */
+  walletConfig: WalletConfig;
+}
+
+export async function launchCustomProviderAndGetWallets<
```

Comment:
I got this name from the [rust-sdk](https://rust.fuel.network/v0.48.0/wallets/test-wallets.html#setting-up-test-wallets). I personally think a more appropriate name would be `setupTestProviderAndWallets`, but just went with their name for similarity's sake.